### DISCUSSION TITLE
Feature: (New Tab)stats regarding all blocked ads data.

### DISCUSSION THREAD

**Discussion by meetpateltech** [ReVanced/revanced-discussions#864] [General]
I know it's possible, Tab that show till date how many blocked ads, how much time saved from this blocking ads in video.
Hope developers see this and make patch for this feature!

**Comment by LisoUseInAIKyrios** [comment]
Something like this could be added to the ReVanced settings (similar to how RYD connection stats are shown when debug is enabled).

The downside is some of the stats will not be very meaningful, such as hiding litho ads (such as ads in the home feed).  This is because scrolling the content off screen and scrolling on screen, it will count the same ad more than once.  So the stats will be inflated and not a true count of how many unique ads were blocked.
